lapack/gonum/dormhr.go

```diff
@@ -4,7 +4,9 @@
 
 package gonum
 
-import "gonum.org/v1/gonum/blas"
+import (
```

Comment:
Can you undo this please.

Reply:
It must be one of the formatting tools that does this. Done.